src/biome/data/sources/readers.py

```diff
@@ -38,10 +35,19 @@ def from_csv(path: Union[str, List[str]], **params) -> DataFrame:
         A `dask.DataFrame`
 
     """
-    return dd.read_csv(path, include_path_column=True, **params)
+    path_list = _get_file_paths(path)
```

Comment:
This is not necessary since dask.dataframe.read_csv can load multiple files 

See : https://docs.dask.org/en/latest/dataframe-api.html#dask.dataframe.read_csv

Reply:
I was worried about the *path* column of the resulting DataFrame.
I just checked and the `include_path_column` parameter in the `read_csv` works:
```python
df = dd.read_csv("*.csv", include_path_column=True)
df.compute().path.value_counts()
# 1.csv    1
# 2.csv    1
# Name: path, dtype: int64
```

But the `dd.read_json` and `dd.read_parquet` methods do not have such an option, so the *path* column would only include the path name with the wildcard
```python
df = dd.read_json("*.json")
df.compute().path.value_counts()
# *.json    2
# Name: path, dtype: int64
```
@frascuchon I will only change the csv reader then, right?

Reply:
Mkay @dcfidalgo 

Yes, just the csv using the default dask behaviour